app/src/main/java/mozilla/lockbox/store/ClipboardStore.kt

```diff
@@ -48,5 +51,16 @@ open class ClipboardStore(
     private fun addToClipboard(label: String, str: String) {
         val clip = ClipData.newPlainText(label, str)
         clipboardManager.primaryClip = clip
+        clearClipboard(clipDataClearDelay, str)
+    }
+
+    private fun clearClipboard(delay: Long, clearStr: String) {
```

Comment:
Rename these variables to fit the intent of the method.

Note that the method is approximately: clear the clipboard after a delay if and only if the clipboard contains this string.

Suggest: 

```kt
fun timedReplaceDirty(dirty: String, clean: String, delay: Int = clipDataClearDelay) {}

fun replaceDirty(dirty: String, clean: String) {}
```

Note:

 * since the class is called `ClipboardStore`, you don't have to have `Clipboard` in every identifier name.
 * you can test the second method pretty directly.
 * since you're calling the clear methods at the same time as the copy, you can restore the clipboard back to its original state after the credentials have been copied. This will require careful bookkeeping, so this may be worth discussing with product before doing this.

Reply:
code updated with changes